File: src/frontend/src/lib/types/lobby.ts
export type Lobby = {
	name: string;
	width: number;
	height: number;
	players: string[];
	mapTiles: number[];
};

export function getTile(mapTiles: number[], width: number, x: number, y: number): number {
	const index = y * width + x;	
    return mapTiles[index];
}

File: src/backend/BotBattle.Api/Models/LobbySpawner/Lobby.cs
using BotBattle.Core;

namespace BotBattle.Api.Models.LobbySpawner;

public class Lobby
{
    public Guid LobbyId { get; set; }
    public int Width { get; set; }
    public int Height { get; set; }
    public Player[] Players { get; set; }
    public int[] ArenaDimension { get; set; }
    public int[] MapTiles { get; set; }
    public event EventHandler<Lobby> LobbyFinished;

    public void OnLobbyFinished()
    {
        LobbyFinished?.Invoke(this, this);
    }
}